File: hello.py
import time
start = time.time()

resultant_file = open('result_python.js', 'w')

with open('angularjs.js') as file:
    data = file.read()

final_data = data.replace('var', 'let')
resultant_file.write(final_data)

resultant_file.close()

print((time.time()-start)*1000)

# 10ms

File: hello.js
var start = new Date()

var fs = require('fs');

const data = fs.readFileSync('./angularjs.js', {encoding:'utf8', flag:'r'});

final_data = data.replace("var", "let");

fs.writeFileSync("result_node.js", final_data); 

var end = new Date() - start

console.log('time consumed:', end)

// 16ms